### DISCUSSION TITLE
How to customize postgres config when self-hosting ?

### DISCUSSION THREAD

**Discussion by actraiser** [supabase/supabase#22669] [Questions]
There is some documentation over at https://supabase.com/docs/guides/platform/custom-postgres-config on how to update the postgres instance when hosting with supabase.co.  To do so, one would use supabase cli using  the "--experimental  postgres-config update"  parameter, but that does not support specifying a --db-url to target a self-hosted instance as far as I can see. 

So my question is, how can i configure those options like max_connections or shared_buffers when self-hosting by running the official docker-compose.yml over at https://github.com/supabase/supabase/tree/master/docker

Thank you and Greets
-act

**Comment by actraiser** [comment]
Replying my own question. Not sure why I overlooked it before, but the docker-compose file clearly starts postgres with a custom command including "- config_file=/etc/postgresql/postgresql.conf" where everything can be customized.

Greets
-act

**Comment by raman04-byte** [comment]
<img width="1651" alt="image" src="https://github.com/supabase/supabase/assets/76096922/f78f56c9-3e36-4183-80ea-5c4cec594faa">
Can you help me in this problem I am facing in Postgres
